### ISSUE TITLE
Front readme incorrectly implies shell expansion support exists

### ISSUE THREAD

**Issue by TomMD** [GuardTime/libksi#10]
In the front readme the line exists:

```
    KSI_PublicationsFile_fromFile(ksi, "~/ksi-publications.bin", &pubFile);
```

Use of tilde implies some shell-expansion like support exists.  Totally possible given things like `wordexp`... but not actually the case.  If a place-holder is desired then I suggest a well named but undefined symbol such as `KSI_PUBLICATIONS_BIN_FILE`.

**Comment by margoraja** [comment]
Will clean up the readme and tutorials.

**Comment by vii5ard** [comment]
Fixed in release [v3.10.1839](https://github.com/guardtime/libksi/releases/tag/v3.10.1839)
